mappings/net/minecraft/client/gui/GuiMainMenu.mapping

```diff
@@ -2,11 +2,23 @@ CLASS bgu net/minecraft/client/gui/GuiMainMenu
 	FIELD A SPLASHES_LOC Lkp;
 	FIELD B LOGO_LOC Lkp;
 	FIELD C PANORAMA_LOC [Lkp;
+	FIELD D outdatedGlTextWidth I
```

Comment:
It's not just a generic warning, it's only used for the outdated GL warning. `warningTextWidth` and such would be fine though, it would just be less precise.